cdap-ui/app/cdap/components/EntityListView/index.js

```diff
@@ -410,7 +423,57 @@ class EntityListView extends Component {
     });
   }
 
+  dismissSplash() {
```

Comment:
@ajainarayanan how will this work on an insecure cluster (no authentication), like a development cluster can be.
If one user dismisses the splash, it'll be dismissed for all other users also?